src/Factories/JavascriptFactory.php

```diff
@@ -13,19 +13,20 @@ class JavascriptFactory
      */
     protected $widgetFactory;
 
-    /**
+       /**
      * Ajax link where widget can grab content.
      *
      * @var string
      */
-    protected $ajaxLink = '/arrilot/load-widget';
-
+	protected $ajaxLink;
+   
     /**
      * @param $widgetFactory
      */
     public function __construct(AbstractWidgetFactory $widgetFactory)
     {
         $this->widgetFactory = $widgetFactory;
+	    $this->ajaxLink = $this->widgetFactory->app->config('laravel-widgets.ajaxLink', 'arrilot/load-widget');
```

Comment:
$this->ajaxLink = $this->widgetFactory->app->config('laravel-widgets.ajax_load_widget_link', '/arrilot/load-widget');

You also need to fix faling tests

Reply:
Please merge this PR, without adding this configuration, 'asyncWidget' functionality won't work.